library/src/main/java/com/getstream/sdk/chat/interfaces/MessageSendManager.java

```diff
@@ -0,0 +1,12 @@
+package com.getstream.sdk.chat.interfaces;
+import com.getstream.sdk.chat.rest.Message;
+
+/**
+ * Interface for Send Message
+ */
+public interface MessageSendManager {
```

Comment:
I would call it just listener, but not manager. An implementation is not supposed to manage anything, but just receive success/error result.